src/libstd/lib.rs

```diff
@@ -331,6 +331,10 @@ extern crate alloc as alloc_crate;
 #[allow(unused_extern_crates)]
 extern crate libc;
 
+#[cfg(any(target_os = "macos", target_os = "ios"))]
+#[allow(unused_extern_crates)]
+extern crate mach;
```

Comment:
I don't think this is necessary since libstd is in the 2018 edition and `mach::...` should "just work" everywhere else.

Reply:
You mean the `#[allow(unused_extern_crates)]` ?

Reply:
Or the whole thing ?

Reply:
I've removed the whole thing, let's see what CI says.